test/bdd/pkg/issuer/issuer_steps.go

```diff
@@ -204,6 +247,61 @@ func (e *Steps) signCredential(didDocVar string) ([]byte, error) {
 	return responseBytes, nil
 }
 
+func (e *Steps) composeAndIssueCred(user string) error {
+	did := e.bddContext.Args[user]
+	log.Infof("DID for signing %s", did)
+
+	if err := bddutil.ResolveDID(e.bddContext.VDRI, did, 10); err != nil {
+		return err
+	}
+
+	req := fmt.Sprintf(composeCredReqFormat, did)
+
+	endpointURL := issuerURL + "/credentials/composeAndIssueCredential"
+
+	resp, err := http.Post(endpointURL, "application/json", //nolint: bodyclose
+		bytes.NewBufferString(req))
+	if err != nil {
+		return err
+	}
+
+	defer bddutil.CloseResponseBody(resp.Body)
+
+	responseBytes, err := ioutil.ReadAll(resp.Body)
+	if err != nil {
+		return fmt.Errorf("failed to read response : %s", err)
+	}
+
+	if resp.StatusCode != http.StatusOK {
+		return fmt.Errorf("got unexpected response from %s status '%d' body %s",
+			endpointURL, resp.StatusCode, responseBytes)
+	}
+
+	log.Infof("vc with proof %s", string(responseBytes))
+
+	signedVCResp := make(map[string]interface{})
+
+	err = json.Unmarshal(responseBytes, &signedVCResp)
+	if err != nil {
+		return err
+	}
+
+	proof, ok := signedVCResp["proof"].(map[string]interface{})
+	if !ok {
+		return errors.New("unable to convert proof to a map")
+	}
+
+	if proof["type"] != "Ed25519Signature2018" {
```

Comment:
TODO for the NIST P256 version?

Reply:
https://github.com/trustbloc/edge-service/issues/new